Nitrox.Test/Server/Serialization/WorldPersistenceTest.cs

```diff
@@ -102,7 +102,6 @@ public void VehicleDataTest(PersistedWorldData worldDataAfter, string serializer
     [DataTestMethod, DynamicWorldDataAfter]
     public void InventoryDataTest(PersistedWorldData worldDataAfter, string serializerName)
     {
-        AssertHelper.IsListEqual(worldData.WorldData.InventoryData.InventoryItems.OrderBy(x => x.ItemId), worldDataAfter.WorldData.InventoryData.InventoryItems.OrderBy(x => x.ItemId), ItemDataTest);
```

Comment:
Tests are missing for:
- InventoryEntity
- InventoryItemEntity
- CrafterMetadata
- PlantableMetadata